web/src/main/java/no/nav/foreldrepenger/web/app/tjenester/forvaltning/praksisutsettelse/FeilPraksisOpprettBehandlingTjeneste.java

```diff
@@ -127,33 +125,50 @@ public void opprettBehandling(Fagsak fagsak) {
+            }
             return;
         }
 
-        var enhet = behandlendeEnhetTjeneste.finnBehandlendeEnhetFra(sisteVedtatte);
-        fagsakLåsRepository.taLås(fagsak.getId());
-        var revurdering = revurderingTjeneste.opprettAutomatiskRevurdering(fagsak, BehandlingÅrsakType.FEIL_PRAKSIS_UTSETTELSE, enhet);
-        behandlingProsesseringTjeneste.opprettTasksForStartBehandling(revurdering);
-        LOG.info("FeilPraksisUtsettelse: Opprettet revurdering med behandlingId {} saksnummer {}", revurdering.getId(), fagsak.getSaksnummer());
+        if (behandlingRepository.hentAbsoluttAlleBehandlingerForFagsak(fagsak.getId()).stream()
+                .anyMatch(b -> b.harBehandlingÅrsak(BehandlingÅrsakType.FEIL_PRAKSIS_UTSETTELSE))) {
+            LOG.info("FeilPraksisUtsettelse: Har allerede hatt feil praksis revurdering for sak {}", fagsak.getSaksnummer());
+        }
+
+        if (!dryrun) {
+            var enhet = behandlendeEnhetTjeneste.finnBehandlendeEnhetFra(sisteVedtatte);
+            fagsakLåsRepository.taLås(fagsak.getId());
+            var revurdering = revurderingTjeneste.opprettAutomatiskRevurdering(fagsak, BehandlingÅrsakType.FEIL_IVERKSETTELSE_FRI_UTSETTELSE, enhet);
+            behandlingProsesseringTjeneste.opprettTasksForStartBehandling(revurdering);
+        }
+
+        LOG.info("FeilPraksisUtsettelse: Opprettet revurdering med saksnummer {}", fagsak.getSaksnummer());
 
     }
 
     private static Trekkdager tapteDager(List<UttakResultatPeriodeEntitet> perioder) {
-        Map<UttakAktivitetGruppering, Trekkdager> trekkdagerPrAktivitet = new LinkedHashMap<>(tapteDagerUtsettelse(perioder));
+        var trekkdagerPrAktivitet = new LinkedHashMap<>(tapteDagerUtsettelse(perioder));
         tapteDagerGradering(perioder).forEach((k,v) -> trekkdagerPrAktivitet.put(k, trekkdagerPrAktivitet.getOrDefault(k, Trekkdager.ZERO).add(v)));
         return trekkdagerPrAktivitet.values().stream().max(Comparator.naturalOrder()).orElse(Trekkdager.ZERO);
     }
 
     private static Map<UttakAktivitetGruppering, Trekkdager> tapteDagerUtsettelse(List<UttakResultatPeriodeEntitet> perioder) {
         return perioder.stream()
-            .filter(p -> FEIL_UTSETTELSE.contains(p.getResultatÅrsak()))
+            .filter(FeilPraksisOpprettBehandlingTjeneste::erAvslåttUtsettelse)
             .map(FeilPraksisOpprettBehandlingTjeneste::tapteDagerUtsettelsePeriode)
             .flatMap(Collection::stream)
             .collect(Collectors.groupingBy(UttakAktivitetTapteDager::grupperingsnøkkel,
                 Collectors.reducing(Trekkdager.ZERO, UttakAktivitetTapteDager::taptedager, Trekkdager::add)));
     }
 
+    private static boolean erAvslåttUtsettelse(UttakResultatPeriodeEntitet p) {
+        if (p.getTom().isBefore(UtsettelseCore2021.IKRAFT_FRA_DATO)) {
+            return false;
+        }
+        return p.isUtsettelse() && PeriodeResultatÅrsak.UtfallType.AVSLÅTT.equals(p.getResultatÅrsak().getUtfallType());
```

Comment:
Trenger vi sjekke på avslått utsettelse? @palfi vet kanskje tilfellene av innvilget utsettelse med trekk (er det noe rundt prematur og pleiepenger)

Reply:
Kan vi ikke fortsatt bare telle de årsakene fra forrige kjøring? PeriodeResultatÅrsak.AVSLAG_UTSETTELSE_PGA_FERIE_TILBAKE_I_TID, PeriodeResultatÅrsak.AVSLAG_UTSETTELSE_PGA_ARBEID_TILBAKE_I_TID

Reply:
Hva tenker vi her? Skal vi sjekke på PeriodeResultatÅrsak og ikke alle utsettelser som er avslått?

Reply:
Er PeriodeResultatÅrsak.AVSLAG_UTSETTELSE_PGA_FERIE_TILBAKE_I_TID, PeriodeResultatÅrsak.AVSLAG_UTSETTELSE_PGA_ARBEID_TILBAKE_I_TID og dekkende?

Reply:
If'n her brukes vel til å telle dager tapt. Den burde vel matche uttrekk sql. Der sjekker vi "not in" og en haug med årsaker + ikke utbetaling

Reply:
Vi trenger alle utsettelser uten utbetaling og med trekkdager fom 1/10-21 - men ikke de som er relatert til første 6 uker eller prematur/pleiepenger (4077)
Antagelig like greit å matche SQL - dvs ikke sjekke på avslått men ta inn de 13 årsakene som ikke skal telles.

Reply:
Gjorde en rask sjekk. Alle aktuelle perioder er avslått. 
Men siden vi skal telle dager og sjekke om >=1) så bør vi ta inn de 13 årsakene og bare telle utsettelser som IKKE er en av de 13. Sjekk på avslått er ikke nødvendig.

Ellers OK og snart klar for test

Reply:
Da har jeg lagt det inn. Her er endringen: https://github.com/navikt/fp-sak/pull/7030/commits/71afbcd9506834072c59638201691baf6c92e690